src/components/Banner/AnonBannerSticky.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { DesktopOnly, MobileOnly } from './../Responsive'
+import BannerDesktop from './BannerDesktop'
+import BannerMobile from './BannerMobile'
+import StickyBannerContent from './StickyBannerContent'
+
+class AnonBannerSticky extends React.PureComponent {
+  state = { isVisible: false }
+
+  componentDidMount () {
+    this.viewportHeight = window.innerHeight
```

Comment:
What is the reason for creating this variable? Why it cannot be accessed directly via `window`?

Reply:
it use every time in callback for scroll. As I understand, reading this property [invoke synchronous reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a)

Reply:
Nice catch 👍.
Speaking of performance, it's good to throttle the scroll event

Reply:
I was thinking about it, I'll add it, ok)